### PR TITLE
DCS-1379 updated returning from temporary absence url

### PR DESCRIPTION
**Pull request by louisenorris**
Note: Contents of confirmTemporaryAbsenceAddedToRoll.njk view page to be added in a future ticket, draft view page included as part of this PR to play out navigation from temporary absence check page.

### CODE REVIEW

File: server/utils/nunjucksSetup.ts [hunk 1]
```diff
@@ -48,6 +49,10 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     return `${array[0][0]}. ${array.reverse()[0]}`
   })
 
+  njkEnv.addFilter('formatName', (name: string) => {
+    return convertToTitleCase(name)
+  })
```
Comment (andrewrlee) [inline server/utils/nunjucksSetup.ts:54]: we tend to handle this on the backend for arrivals - we should probably do the same for temp absences

File: server/services/temporaryAbsencesService.ts [hunk 2]
```diff
@@ -13,4 +13,14 @@ export default class TemporaryAbsencesService {
     const temporaryAbsences = await this.welcomeClientFactory(token).getTemporaryAbsences(agencyId)
     return temporaryAbsences.sort(compareByFullName)
   }
+
+  public async getTemporaryAbsence(agencyId: string, prisonNumber: string): Promise<TemporaryAbsence> {
+    const token = await this.hmppsAuthClient.getSystemClientToken()
+    const temporaryAbsence = await this.welcomeClientFactory(token).getTemporaryAbsence(agencyId, prisonNumber)
+    return {
+      ...temporaryAbsence,
+      firstName: convertToTitleCase(temporaryAbsence.firstName),
```
Comment (andrewrlee) [inline server/services/temporaryAbsencesService.ts:22]: formatting twice?

Comment (louisenorris) [reply server/services/temporaryAbsencesService.ts:22]: Agree it isn't great - seemed less code than formatting in both the service get methods but I guess I could have extracted a private formatting method there... will sort in the backend instead as suggested - thank you

Comment (louisenorris) [reply server/services/temporaryAbsencesService.ts]: Removed on latest commit

File: server/utils/nunjucksSetup.ts (hunk 1 above)
Comment (louisenorris) [reply server/utils/nunjucksSetup.ts]: Removed on latest commit

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**